core_scheduled_rules/vpc_dns_tunneling_rule.py

```diff
@@ -0,0 +1,2 @@
+def rule(event):  # pylint: disable=unused-argument
+    return True
```

Comment:
where would suggest putting this file? This also leads to a few of my questions above...

> directory structure we would like to go with for scheduled_rules/queries

Should _all_ scheduled rules be in a single dir? and _all_ queries be in a single dir? Should we separate based on pack membership? Should we... do something else?

Reply:
Well right now we differentiate based roughly on logtype/resource type. So `aws_cloudtrail_rules`, `aws_cloudtrail_policies`, `okta_rules`, etc. I'd suggest something like `aws_queries`. And I would put the corresponding scheduled rules in the same directory as  the scheduled query, since they're not particularly useful without them.